src/overlays/actors/ovl_Demo_Gj/z_demo_gj.c

```diff
@@ -96,241 +84,1326 @@ static ColliderCylinderInitType1 sCylinderInit = {
+    func_8097911C(this, globalCtx, &D_06003190);
+}
+
+void func_8097AB84(DemoGj* this, GlobalContext* globalCtx) {
+    DemoGj_InitSetIndexes(this, globalCtx, 7, 8, &D_06003AF0);
+}
+
+void func_8097ABB4(DemoGj* this, GlobalContext* globalCtx) {
+    Actor_MoveForward(&this->dyna.actor);
+
+    this->rotationVec.x += (s16)(kREG(15));
+    this->rotationVec.y += (s16)(kREG(14) + 0x3E8);
+    this->rotationVec.z += (s16)(kREG(13) + 0xBB8);
+
+    DemoGj_Reflect(this, globalCtx);
+}
+
+static Vec3f D_8097BEA0 = { -6.0f, 1053.0f, -473.0f };
+
+void func_8097AC30(DemoGj* this, GlobalContext* globalCtx) {
+    u32 gameplayFrames;
+
+    if (!this->flag1) {
+        gameplayFrames = globalCtx->gameplayFrames % 3;
+
+        if (1) {}
+        if (gameplayFrames == 2) {
+            if (!globalCtx->gameplayFrames) {}
+            func_80978AFC(globalCtx, &D_8097BEA0, 300.0f);
+        }
+
+        func_80979F9C(this);
+    }
+}
+
+void func_8097AC9C(DemoGj* this, GlobalContext* globalCtx) {
+    if (func_80979818(this, globalCtx)) {
+        DemoGj_SetupMovement(this, globalCtx);
+        this->updateIndex = 14;
+        this->drawIndex = 15;
+    }
+}
+
+void func_8097ACE8(DemoGj* this, GlobalContext* globalCtx) {
+    if (func_8097983C(this, globalCtx)) {
+        Actor_Kill(&this->dyna.actor);
+    }
+}
+
+void DemoGj_Update7(DemoGj* this, GlobalContext* globalCtx) {
+    DemoGj_FindGanon(this, globalCtx);
+    func_8097AC9C(this, globalCtx);
+}
+
+void DemoGj_Update14(DemoGj* this, GlobalContext* globalCtx) {
+    func_8097ABB4(this, globalCtx);
+    func_8097ACE8(this, globalCtx);
+}
+
+void DemoGj_Draw8(DemoGj* this, GlobalContext* globalCtx) {
+    DemoGj_DrawCommon(this, globalCtx, &D_06003710);
+}
+
+void DemoGj_Draw15(DemoGj* this, GlobalContext* globalCtx) {
+    func_8097911C(this, globalCtx, &D_06003710);
+}
+
+void func_8097ADC0(DemoGj* this, GlobalContext* globalCtx) {
+    DemoGj_InitSetIndexes(this, globalCtx, 0, 1, &D_06001B70);
+}
+
+void DemoGj_Update0(DemoGj* this, GlobalContext* globalCtx) {
+    DemoGj_FindGanon(this, globalCtx);
+    func_8097A07C(this, globalCtx);
+    func_8097A2B4(this, globalCtx);
+    func_8097AC30(this, globalCtx);
+}
+
+void DemoGj_Draw1(DemoGj* this, GlobalContext* globalCtx) {
+    DemoGj_DrawCommon(this, globalCtx, &D_06000DC0);
+}
+
+void func_8097AE5C(DemoGj* this, GlobalContext* globalCtx) {
+    DemoGj_InitSetIndexes(this, globalCtx, 15, 0, NULL);
+    DemoGj_Cylinder_Init(this, globalCtx, &this->cylinders[0], &sCylinderInit1);
+    DemoGj_Cylinder_Init(this, globalCtx, &this->cylinders[1], &sCylinderInit1);
+    DemoGj_Cylinder_Init(this, globalCtx, &this->cylinders[2], &sCylinderInit1);
+}
+
+void func_8097AEDC(DemoGj* this, GlobalContext* globalCtx) {
+}
+
+void func_8097AEE8(DemoGj* this, GlobalContext* globalCtx) {
+    ColliderCylinder* cylinders_0 = &this->cylinders[0];
+    ColliderCylinder* cylinders_1 = &this->cylinders[1];
+    ColliderCylinder* cylinders_2 = &this->cylinders[2];
```

Comment:
to stick to our camel case styling id remove the underscore. `cylinder1` `cylinder2` etc, it doesnt need to be plural